cmd/achcli/reformat.go

```diff
@@ -42,13 +42,15 @@ func reformat(as string, filepath string) error {
 }
 
 func readIncomingFile(path string) (*ach.File, error) {
-	if file, err := readJsonFile(path); file != nil && err == nil {
+	file, err := readJsonFile(path)
+	if file != nil && err == nil {
 		return file, nil
 	}
-	if file, err := readACHFile(path); file != nil && err == nil {
+	file, err = readACHFile(path)
+	if file != nil && err == nil {
 		return file, nil
 	}
-	return nil, fmt.Errorf("unable to read %s", path)
+	return nil, fmt.Errorf("unable to read %s:\n %v", path, err)
```

Comment:
```suggestion
	return nil, fmt.Errorf("unable to read %s:\nError: %v", path, err)
```
Not necessary, but could help readability

Reply:
This seems to be already done by the `err`:

```
$ achcli -reformat json test/testdata/invalid-two-micro-deposits.ach
ERROR: unable to read test/testdata/invalid-two-micro-deposits.ach:
 line:17 record:Batches *ach.BatchError batch #2 (PPD) TotalDebitEntryDollarAmount calculated 0 is out-of-balance with batch control 44
```

Reply:
ah you're right, thank you